README.md

```diff
@@ -25,25 +25,36 @@ estimated backfill time @ 50rps = 24-36 hours on M1 Macbook (~10x speedup)
   - (possible) continued backwards compatibility with subgraph
   - support indexing subset of data, i.e. only domains under parent node
 
-### todo
+#### known bugs
+
+- [ ] there's an account `0x` in the database, which is definitely a bug — perhaps all addresses should go through `viem/getAddress` before being handled (presumably ponder did this inthe background...)
```

Comment:
It would be curious to see the source event causing this `0x` account to be created. We might learn something interesting or catch an interesting bug.